docs/data-integrations/destinations/adapt-existing-destination.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: "Adapt an existing destination"
```

Comment:
Do you intent to add this file? Is it ready for review?

Reply:
Nope— tommy actually said to drop that one. I'll remove